Airwallex/Core/Sources/AWXSession.h

```diff
@@ -44,6 +44,16 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, strong, nullable) AWXApplePayOptions *applePayOptions;
 
+/**
+Whether AWXPaymentMethodListViewController should show stored card.
+ */
+@property (nonatomic) BOOL hidePaymentConsents;
+
+/**
+Whether AWXPaymentMethodListViewController should show stored card.
```

Comment:
Need to correct this comment.

Reply:
We can let user refer to API docs otherwise they might config the name incorrectly, also let's tolerate case insensitivity.
`https://www.airwallex.com/docs/api#/Payment_Acceptance/Config/_api_v1_pa_config_payment_method_types/get`